### PR TITLE
fix(curriculum):  Step 103 Pyramid Generator Text

### LINKED ISSUE #55158: Step 103 Pyramid Generator doesn't say where to write the code
### Describe the Issue

There is no mention of where to write the code, but the tests expect the two line right below the `numbers` array declaration.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-introductory-javascript-by-building-a-pyramid-generator/step-103

### Your code

```
const numbers = [1, 2, 3];

const unshifted = numbers.unshift(5);
console.log(unshifted);
console.log(numbers);

const shifted = numbers.shift();
console.log(shifted);


```

### Expected behavior

I would expect to know where to write, or for code to be accepted anyway

### Screenshots

_No response_

### System

=

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mpdillingerOSU**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes  #55158

First-time contributor! I have followed the exact recommended change -as set by @Ksound22 within the discussion of the issue. If I did anything incorrectly, please let me know! I want to continue contributing to this project in the future.

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
No, sorry, you made changes to the arabic file

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
No, sorry, you made changes to the arabic file

**Review by mpdillingerOSU — COMMENTED** [review]
Thank you for the note! My fault for not realizing that different languages still shared the same code file name when using the search function. Will remember that for the future!

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f4e74f7fd3f4a99ac2e50.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ numbers.shift();
 
 The `numbers` array would be `[2, 3]`.
 
-Declare a `shifted` variable, assign it the result of calling `.shift()` on your `numbers` array, and print the variable.
+Directly below the declaration of `numbers`, declare a `shifted` variable, and assign it the result of calling `.shift()` on your `numbers` array. Then, on the next line, print the variable.
```
Comment (Ksound22) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f4e74f7fd3f4a99ac2e50.md:21]: We can make the text simpler a bit:

```suggestion
Directly below your `numbers` array, declare a `shifted` variable and assign it the result of calling `.shift()` on the `numbers` array. On the next line, log the `shifted` variable to the console.
```

**Review by Sembauke — APPROVED** [review]

**Pull request merged**